server/protocols/webadmin-cli/src/test/resources/lmtpserver.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
```

Comment:
Can this file be ommitted?

Reply:
If i delete these xml file, my inmemory james/unit test will be failed. I don't know exactly which files to delete. Maybe I need some help on this?